modules/gdscript/gdscript_parser.h

```diff
@@ -562,6 +562,7 @@ class GDScriptParser {
 				ENUM,
 				ENUM_VALUE, // For unnamed enums.
 				GROUP, // For member grouping.
+				TOOL_BUTTON,
```

Comment:
I don't think `TOOL_BUTTON` is necessary except semantically. We could reuse `GROUP` and rename it to `META`, `PSEUDO`, `FAKE` or something like that.

Reply:
Do you have any changes you want that would be suitable for this PR? I am not comfortable globbing together the refactoring of `GROUP` into `META` here, although it would make a good follow-up PR.

Reply:
I don't think it's worth a separate PR, it's not much code. You can just not add `TOOL_BUTTON` next to each `GROUP`, and instead rename `GROUP` to `META`. See the review below.